src/id-card/index.html

```diff
@@ -93,19 +93,21 @@
                         luxon.Settings.defaultLocale = LOCALE_NZ;
 
                         const now = luxon.DateTime.now().toUTC();
-                        const expiryDate = luxon.DateTime.now().plus({years: 10}).toUTC();
+                        const validity = now.plus({months: 1}).toUTC();
```

Comment:
Can I ask why 1 month validity?

Reply:
Yes worth a discussion. I mentioned 1 month as an idea for a business model might be a monthly subscription to have access to the digital version of the physical card. It's really just a chance for us to play with and test this particular feature of IRMA we haven't used yet.

Reply:
Currently the agreed Acceptance Criteria for https://app.asana.com/0/1171124854073033/1200078086990964. But we can discuss and change it quite easily as Cam explained.